camera/rpi-files/main.py

```diff
@@ -12,8 +12,11 @@
 import time
 
 import openapi_client
+from data_batcher import DataBatcher
 from dotenv import load_dotenv
-from openapi_client.rest import ApiException
+from people_counter import PeopleCounter
+
+BATCH_SECONDS = 3
```

Comment:
we should take into account that the deploy server has limited number of requests allowed and we should not send requestes without needs so 3s time as for me seems too short (it may be one minute or even more - to be discussed). Lets introduce maybo also somethin like batching time that is gonna be defined on server not to grab data for the whole time if not needed

Reply:
We will need to hit the server repeatedly to confirm the device is still online. I set batching each 3 seconds for testing, but I think realistically it will something around 15 seconds. I'll test when devices are set up.